### PR TITLE
Check pawc and paw

### LINKED ISSUE #4312: Error when PAW and PAWC equal to zero
MeanWTF equal to NA is PAW and PAWC equal to zero in the soil  when ll equals to dul.

### PR DESCRIPTION
**Pull request by byzheng**
resolves #4312

### CODE REVIEW

File: Models/Plant/Organs/Root.cs [hunk 1]
```diff
@@ -395,7 +395,10 @@ public double WaterTensionFactor
                         double[] pawc = Z.soil.PAWC;
                         Biomass[] layerLiveForZone = Z.LayerLive;
                         for (int i = 0; i < Z.LayerLive.Length; i++)
-                            MeanWTF += layerLiveForZone[i].Wt / liveWt * MathUtilities.Bound(2 * paw[i] / pawc[i], 0, 1);
+                            if (paw[i] < pawc[i] & paw[i] > 0 & pawc[i] > 0)
```
Comment (hol430) [inline Models/Plant/Organs/Root.cs]: Should these bitwise AND operators be logical AND?
```suggestion
                            if (paw[i] < pawc[i] && paw[i] > 0 && pawc[i] > 0)
```

**Comment by byzheng** [comment]
@hol353 Anything I can do for this pull request? It seems there are big changes of results for the small changes of codes.

**Comment by hol353** [comment]
Your changes obviously had a bit impact. Were you expecting such a big change?

**Comment by byzheng** [comment]
I don't expect a big change. This bug only has impact growth when DUL equals to LL in the soil profile.

**Comment by hol353** [comment]
You'll have to investigate why you are getting major changes. Pick a simulation where you are getting big diffs and run it before and after your change and look at daily outputs to determine what is different.

**Comment by BenAbabaei** [comment]
Try paw[i] **<=** pawc[i] in the if-statement.

**Comment by byzheng** [comment]
retest this please jenkins

**Comment by BenAbabaei** [comment]
If you don't include **paw[i] <= pawc[i]** in the if-statement, it ignores the laters that are fully moist.

**Comment by hol353** [comment]
retest this please jenkins

**Pull request merged**